src/api/content.ts

```diff
@@ -2,17 +2,17 @@ import crypto from 'crypto';
 import fm from 'front-matter';
 import fs from 'fs';
 import _glob from 'glob';
-import debounce from 'lodash.debounce';
 import path from 'path';
 import util from 'util';
 import logger from '../config/logger';
-import { ILemma } from '../models/lemma';
-import { ITopic } from '../models/topic';
+import { ILemma } from '../models/Lemma';
+import Topic, { ITopic } from '../models/Topic';
 import * as db from './db';
 import { parseBody } from './parser';
 import TaskQueue from './TaskQueue';
 
 const glob = util.promisify(_glob);
+const removeFile = util.promisify(fs.unlink);
```

Comment:
Most `fs.` functions now also have a promise version; you no longer to promisify them:

https://nodejs.org/dist/latest-v12.x/docs/api/fs.html#fs_fs_promises_api

Reply:
Ah yes, I forgot! thanks for the rescue, I'm still an old fasion
I will make the change and puush it again once the review completed, for not having many commits since it work the same :)